.github/workflows/build-images.yml

```diff
@@ -364,7 +364,19 @@ jobs:
         else
           make docker-build-operator
         fi
-
+    - name: Optionally push operator
+      if: ${{ github.event_name != 'pull_request' && inputs.operator_image == '' }}
+      run: |
+        # Check if image exists in private repo, if not exist, push to repo
+        export OPERATOR_IMG=${{ steps.operator_image.outputs.value }}
```

Comment:
But what will be the value of `OPERATOR_IMG`? You need to make sure the name we picked is unique and contains the said repo.
And what repo is it?

Reply:
This part does not apply anymore, moved the logic to the existing step. Ref: https://github.com/vertica/vertica-kubernetes/pull/928#discussion_r1767095110